registrations/models.py

```diff
@@ -319,6 +321,22 @@ class SignUpGroup(CreatedModifiedBaseModel, SignUpMixin):
         default=None,
     )
 
+    user_data = models.ForeignKey(
```

Comment:
I see we're assigning this user_data to be the same as the `created_by` user, which raises a question; Is this relation necessary and could we use just the `created_by` relation? In the gdpr api we could fetch the sign up data based with the created_by field 🤔

Reply:
I was thinking this same while implementing this, but then thought that here could be two benefits, though pretty minor ones, to having a model and a relation for the GDPR data:
- looking in the database only for whatever reason like data analysis, it is easy to recognize tables that contain GDPR data when they have a field like this relation
- deletion of the GDPR data happens through the cascade behavior so there is no need to introduce additional code for deleting the separate tables' rows

Reply:
Agree that the CASCADE is handy - of course we could have the cascading on the created_by user but perhaps its reasonable to have it like this for future use cases etc. 👌